cmd/root.go

```diff
@@ -11,13 +11,8 @@ import (
 // rootCmd represents the base command when called without any subcommands
 var rootCmd = &cobra.Command{
 	Use:   "opcap",
-	Short: "A brief description of your application",
-	Long: `A longer description that spans multiple lines and likely contains
-examples and usage of using your application. For example:
-
-Cobra is a CLI library for Go that empowers applications.
-This application is a tool to generate the needed files
-to quickly create a Cobra application.`,
+	Short: "A cli tool to gauge capabilities of operators available on OpenShift's operator hub.",
```

Comment:
I would either capitalize CLI since it's an initialism, or more preferably just write out the whole phrase so there's no ambiguity. I would also add "the" after "to gauge" so it's "to gauge the capabilities."

Also, and this is actually a problem with the first line of our README as well, opcap isn't just for "operator hub," right? It works with Red Hat marketplace as well, or custom catalogs. So I think we need to change the "on OpenShift's operator hub" to something more generic. And if we do end up mentioning "operator hub" it should be a proper noun styled like the title of the website, OperatorHub.io.